packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -527,29 +527,33 @@ export interface DocsQAProps {
   corpus: Corpus;
 
   /**
-   * The query to answer.
+   * The question to answer.
    */
   question: string;
 
   /**
    *
-   * The maximum number of documents to return.
+   * When processing a DocsQA query, the most relevent chunks are presented to the model. This
+   * field limits the number of chunks to consider. If this value is too large, the size
+   * of the chunks may exceed the token limit of the model.
    */
-  limit?: number;
+  chunkLimit?: number;
 
   /**
    * The component used to format document chunks when they're presented to the model.
+   * This can be used to present the model with some metadata about the chunk, in a format
+   * that is appropriate to the type of document.
    *
    * ```tsx
-   *  function MyDocsComponent({ doc }: { doc: ScoredChunk }) {
+   *  function FormatChunk({ chunk }: { chunk: ScoredChunk }) {
```

Comment:
Minor, but typically we'd name a component declaratively, not imperatively. Like, `Timestamp`, rather than `ShowTime`. `UserAvatar` rather than `ShowUserImage`. etc.

So maybe here it's `ChunkTemplate`? Or `ChunkView`?